src/widgetastic_patternfly4/wizard.py

```diff
@@ -0,0 +1,286 @@
+from widgetastic.widget import Text
+from widgetastic.widget import View
+from widgetastic.widget import Widget
+from widgetastic.widget import WTMixin
+
+from widgetastic_patternfly4 import Button
+
+
+class TitledContentMixin(WTMixin):
+
+    TITLE_LOCATOR = ".//h1"
+
+    @property
+    def title(self):
+        return self._get_value_if_displayed(self.TITLE_LOCATOR)
+
+    @property
+    def subtitle(self):
+
+        SUBTITLE_LOCATOR = f"{self.TITLE_LOCATOR}//following-sibling::*"
+        return self._get_value_if_displayed(SUBTITLE_LOCATOR)
+
+    def _get_value_if_displayed(self, locator):
+
+        widget = Text(self, locator)
+
+        if widget.is_displayed:
+            return widget.text
+        else:
+            self.logger.warning(
+                "widget with locator '%s' not found, returning empty string value", locator
+            )
+            return ""
+
+
+class WizardSteps(View):
+
+    ROOT = ".//nav[@class='pf-c-wizard__nav']"
+    STEPS_LOCATOR = ".//li[@class = 'pf-c-wizard__nav-item']"
+    CURRENT_STEP = f"{STEPS_LOCATOR}/button[@aria-current='page']"
+
+    @property
+    def steps(self):
+        return [self.browser.text(elem) for elem in self.browser.elements(self.STEPS_LOCATOR)]
+
+    @property
+    def active_step(self):
+        return self.browser.element(self.CURRENT_STEP).text
+
+    def select_step(self, step_name):
+        button_locator = self.STEPS_LOCATOR + f'/button[text()="{step_name}"]'
+        self.browser.element(button_locator).click()
+
+
+class WizardHeaderView(View, TitledContentMixin):
+
+    ROOT = ".//div[@class='pf-c-wizard__header']"
+    TITLE_LOCATOR = ".//h2"
+    CLOSE_LOCATOR = ".//button[contains(@class, 'pf-c-wizard__close')]"
+    close_button = Button(locator=CLOSE_LOCATOR)
+
+
+class WizardFooterView(View):
+
+    ROOT = (
+        ".//div[@data-ouia-component-type='PF4/ModalContent']"
+        "//footer[contains(@class, 'pf-c-wizard__footer')]"
```

Comment:
I don't think this is a good design decision . We formerly had here 
```
.//footer[@class='pf-c-wizard__footer']
```
Which applied to **any** wizard footer class. I understand the problem you're trying to solve is to make it work under the specific "wizard on modal" scenario from the test page, but I'd better add a mechanism to switch between a wizard in modal or a regular wizard embedded in page. Hardcoding this to the ModalContent limits it's usage.